docs/Dialects/Comb/RationaleComb.md

```diff
@@ -68,7 +68,13 @@ TODO: describe why add/xor/or are variadic
 
 ### Operators carry signs instead of types
 
-TODO: describe why we have divu/divs but not addu/adds, and not sint vs uint.
+Operators, in LLVM-2.0 style, which have consistent behavior in module 
+arithmetic with respect to signedness are not modeled with sign.  `comb` 
+operates on signless types with signless operations.  This is in accordance
+with LLVM's approach.
```

Comment:
While this is obvious to those who know the history, is there either a short explanation or a link to MLIR's section on signless types (https://mlir.llvm.org/docs/Rationale/Rationale/#integer-signedness-semantics) which explains some of the backstory. I didn't audit this rationale to see if this was already mentioned.